README.md

```diff
@@ -1,16 +1,17 @@
-# Voice Diagnostics SDK
-The Voice Diagnostics SDK provides functions to test input and output audio devices, and the functionality to test network bandwidth requirements towards Twilio’s servers.
+# RTC Diagnostics SDK
+The RTC Diagnostics SDK provides functions to test input and output devices, and the functionality to test network bandwidth requirements towards Twilio’s servers.
 
-This SDK requires the use of Twilio NTS to perform the network tests. Using NTS will incur charges as per [NTS pricing](https://www.twilio.com/stun-turn/pricing).
+This SDK requires the use of Twilio NTS or your own STUN/TURN servers to perform the network tests. Using NTS will incur charges as per [NTS pricing](https://www.twilio.com/stun-turn/pricing).
```

Comment:
Add "Using Twilio NTS will mirror how Twilio uses STUN/TURN in Programmable Video when connecting to a Twilio Video Room."